ChargeApp/spec/views/charges/index.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe "charges/index", type: :view do
+#   before(:each) do
+#     assign(:charges, [
+#       Charge.create!(
+#         :paid => false,
+#         :amount => 1,
+#         :currency => "Currency",
+#         :refunded => false,
+#         :customer_id => 2,
+#         :disputed => false
+#       ),
+#       Charge.create!(
+#         :paid => false,
+#         :amount => 1,
+#         :currency => "Currency",
+#         :refunded => false,
+#         :customer_id => 2,
+#         :disputed => false
+#       )
+#     ])
+#   end
+# 
+#   it "renders a list of charges" do
+#     render
+#     assert_select "tr>td", :text => false.to_s, :count => 2
+#     assert_select "tr>td", :text => 1.to_s, :count => 2
+#     assert_select "tr>td", :text => "Currency".to_s, :count => 2
+#     assert_select "tr>td", :text => false.to_s, :count => 2
+#     assert_select "tr>td", :text => 2.to_s, :count => 2
+#     assert_select "tr>td", :text => false.to_s, :count => 2
+#   end
```

Comment:
I haven't tested the view before because I was told it wasn't stable enough, but I left the generated boiler code here if it was more appropriate for this circumstance.